packages/config/src/class-config.php

```diff
@@ -8,10 +8,9 @@
 namespace Automattic\Jetpack;
 
 use Automattic\Jetpack\Connection\Manager;
-use Automattic\Jetpack\JITM;
```

Comment:
Is this intentional? I see JITM used in line 230

Reply:
`JITM` is located in the same name namespace as `Config` (`Automattic\Jetpack`), as well as `Terms_Of_Service`. So we don't really need to explicitly import those classes.